torch/csrc/autograd/profiler.cpp

```diff
@@ -340,3 +347,45 @@ void RecordProfile::processEvents(const std::vector<Event*>& events) {
 }
 
 }}}
+
+
+namespace {
+
+void callBoxedWorkaround(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+  // This should just be op.callBoxed(stack), but that doesn't work for all ops yet.
+  // Note: If op.callBoxed(stack) works for you, then that is preferrable because
+  // it's much faster and doesn't come with a dependency on JIT code.
+  // Instead, we take a path through the JIT operator registry, which has a boxed
+  // calling mechanism that works for all ops from native_functions.yaml.
+
+  auto s = c10::Symbol::fromQualString(op.schema().name());
+  auto operators = torch::jit::getAllOperatorsFor(s);
+  // Find the exact match
+  std::shared_ptr<torch::jit::Operator> jit_op;
+  for (const auto& candidate_op : operators) {
+    auto candidate_schema = candidate_op->schema();
+    // NB: this is a VERY slow equality test
+    if (candidate_schema == op.schema()) {
+      jit_op = candidate_op;
+      break;
+    }
+  }
+  TORCH_INTERNAL_ASSERT(jit_op);
+
+  auto offset = jit_op->getOperation()(*stack);
+  TORCH_INTERNAL_ASSERT(offset == 0);
+}
+
+}  // namespace
+
+// void profile_wrapper(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+//   c10::impl::ExcludeDispatchKeyGuard key_guard(c10::DispatchKey::ProfilerId);
+//   RECORD_FUNCTION(op.schema().name(), *stack, torch::autograd::Node::peek_at_next_sequence_nr());
+//   callBoxedWorkaround(op, stack);
+// }
+
+// auto registry = c10::Dispatcher::singleton()
+//   .registerBackendFallbackKernel(
+//     c10::DispatchKey::ProfilerId,
+//     c10::KernelFunction::makeFromBoxedFunction<&profile_wrapper>()
```

Comment:
I actually agree with @smessmer's earlier comment, which is that we can keep the backend fallback to implement profiling for user-defined operators that don't have codegenned wrappers. It doesn't interfere with more specific unboxed implementation you are codegenning